ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__amd64-nightly.yaml

```diff
@@ -2566,6 +2566,9 @@ tests:
   cron: 0 16 7 * *
   steps:
     cluster_profile: ibmcloud-qe
+    env:
+      FEATURE_GATES: '[ClusterAPIInstall=true]'
```

Comment:
so do we have blockers on 

- dedicatedhost
- private

installations? I see CAPI is not enabled for those profiles?

Reply:
now private, dedicatedhost, disconnect, byo-kms have bugs in CAPI. 
I enable CAPI in others' profiles to check the e2e test